app/api/users.py

```diff
@@ -284,6 +290,16 @@ def before_update_object(self, user, data, view_kwargs):
                     {'source': ''}, "You are not authorized to update this information."
                 )
 
+        if (
+            not has_access('is_admin')
+            and data.get('is_verified') is not None
```

Comment:
Now, there is one catch left. If the user is not admin, and they sent `None`, what will happen. Please check

Reply:
f the user sends None, then there will be a Validation error (Not a Valid Boolean), doesn't matter if the user is admin or not.
![image](https://user-images.githubusercontent.com/27550875/93991078-81f77500-fda9-11ea-9add-006c0c332425.png)